### DISCUSSION TITLE
How is concatenate different from merge in LosslessCut?

### DISCUSSION THREAD

**Discussion by slrslr** [mifi/lossless-cut#1774] [Q&A] [answered]
I have recent or previous version of the LosslessCut AppImage and i do not see concatenate option yet in it.
Just merge segments.
Is there any description or images to see the difference between merge and concatenate options please?

ChatGPT says: "Merged videos play the clips one after another, while concatenated videos show multiple clips on the screen simultaneously."

**Comment by mifi** [comment, marked as answer]
Merge/concat is the same in losslesscut. Some programs use the term concat and others use merge, so losslesscut also does that.

i think chatgpt is drunk because what he said is not correct. Concatenation afaik always means one after the other (not simultaenously).

**Reply by hrxn** [reply]
Yes, @mifi is entirely correct here, concatenation means chaining together end-to-end, *definitely* not compositing anything simultaneously whatsoever, and ChatGPT is indeed drunk. Or broken. In general.
